### PR TITLE
fix for #11;

### PR DESCRIPTION
**Pull request by chripard**
Please review before merge.

### CODE REVIEW

File: src/main/java/org/repodriller/persistence/csv/CSVFile.java [hunk 1]
```diff
@@ -47,11 +48,13 @@ public CSVFile(String path, String name, boolean append) {
 
 	@Override
 	public synchronized void write(Object... line) {
-		
+		String field;
```
Comment (mauricioaniche) [inline src/main/java/org/repodriller/persistence/csv/CSVFile.java]: this variable can be put inside the for loop, right? Then, we make sure no one is gonna use it afterwards!

File: src/main/java/org/repodriller/persistence/csv/CSVFile.java [hunk 2]
```diff
@@ -47,11 +48,13 @@ public CSVFile(String path, String name, boolean append) {
...
 		boolean first = true;
 		for(Object o : line) {
 			if(!first) ps.print(",");
-			ps.print(o);
+			field = o.toString();
+			field = StringEscapeUtils.escapeCsv(field);
```
Comment (mauricioaniche) [inline src/main/java/org/repodriller/persistence/csv/CSVFile.java]: Very nice! I didn't know this `escapeCsv`!

**Comment by chripard** [comment]
Thank you Mauricio! I was a little dazed and i didn't notice that the method was public.
Hope it's ok now.

File: src/main/java/org/repodriller/persistence/csv/CSVFile.java (hunk 2 above)
Comment (chripard) [inline src/main/java/org/repodriller/persistence/csv/CSVFile.java]: Thank you Mauricio! I was a little dazed and i didn't notice that the method was public.
Hope it's ok now.

**Pull request merged**